lib/govuk_publishing_components/presenters/image_card_helper.rb

```diff
@@ -75,6 +100,45 @@ def context
       def description
         content_tag(:div, @description, class: "gem-c-image-card__description") if @description
       end
+
+      def youtube_href
+        "https://www.youtube.com/watch?v=#{@youtube_video_id}"
+      end
+
+      def youtube_thumbnail_url
+        "https://img.youtube.com/vi/#{@youtube_video_id}/maxresdefault.jpg"
+      end
+
+      def youtube_thumbnail
+        figcaption_class = "gem-c-image-card__youtube-thumbnail-container-text govuk-link"
+        figcaption_class << " brand__color" if @brand
+        thumbnail = content_tag(:div, class: "gem-c-image-card__youtube-thumbnail-image-container") do
+          image_tag(
+            youtube_thumbnail_url,
+            class: "gem-c-image-card__image gem-c-image-card__youtube-thumbnail-image",
+            alt: "",
+            loading: @image_loading,
+          )
+        end
+        thumbnail + content_tag(:figcaption, @image_alt, class: figcaption_class)
```

Comment:
Not 100% sure this will work, but I think you can avoid declaring thumbnail and streamline the end of this method by doing

```Ruby
def youtube_thumbnail
  figcaption_class = "gem-c-image-card__youtube-thumbnail-container-text govuk-link"
  figcaption_class << " brand__color" if @brand
  content_tag(:div, class: "gem-c-image-card__youtube-thumbnail-image-container") do
    image_tag(
      youtube_thumbnail_url,
      class: "gem-c-image-card__image gem-c-image-card__youtube-thumbnail-image",
      alt: "",
      loading: @image_loading,
    )
    content_tag(:figcaption, @image_alt, class: figcaption_class)
  end
end
```

Reply:
In this case I need to render the figcaption outside of the thumbnail image container for styling purposes. Unless I split thumbail into a seperate helper method and then just have `youtube_thumbnail_image + content_tag(:fig_caption...)`